cpp/src/io/parquet/page_enc.cu

```diff
@@ -431,6 +438,7 @@ __global__ void __launch_bounds__(128)
       ck_g.bfr_size           = page_offset;
       ck_g.page_headers_size  = page_headers_size;
       ck_g.max_page_data_size = max_page_data_size;
+      if (not comp_page_sizes.empty()) ck_g.compressed_size = comp_page_offset;
```

Comment:
Another place where I worry about uninitialized data. Do we need to set this to 0 if we don't have page sizes?

Reply:
Same answer. https://github.com/rapidsai/cudf/pull/11066#discussion_r891672148